lib/inherited_class_var.rb

```diff
@@ -90,14 +92,15 @@ def inherited_custom_class(accessor_method_name, base_parent_class)
     # @return [Object] a class variable merged across ancestors until inherited_class_module
     def inherited_class_var(variable_name, default_value, merge_method)
       class_cache(variable_name) do
-        value = default_value
+        current_value = default_value
+
+        ancestor_values = inherited_ancestors.map { |ancestor| ancestor.instance_variable_get(variable_name) }.compact
 
-        inherited_ancestors.each do |ancestor|
-          ancestor_value = ancestor.instance_variable_get(variable_name)
-          value = ancestor_value.public_send(merge_method, value) if ancestor_value.present?
+        ancestor_values.each do |ancestor_value|
+          current_value = current_value.public_send(merge_method, ancestor_value)
         end
 
-        value
+        current_value
```

Comment:
hmm... the library hasn't changed. isn't the implementation mathematically the same here? (just different ordering/syntax)